FD/Finite Differencing.ipynb

```diff
@@ -201,9 +210,8 @@
     "    x, u_initial, dx = initialize(nx, L)\n",
     "    u = u_initial.copy()\n",
     "    \n",
-    "    # Time-stepping loop\n",
```

Comment:
I think it is better to do not add  comment in the mid of the function for formatting purposes.